From e3d8b8a4b0f07b1f0b8bc8bf08894c0336c88ed8 Mon Sep 17 00:00:00 2001
From: Victor Gu <xigu@marvell.com>
Date: Fri, 2 Sep 2016 00:52:18 +0800
Subject: [PATCH 0456/1345] PCI: Aardvark: remove command queue definition

commit  f821a69e4337de648b4b938169e780cfc69c0bb8 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

The command queue feature is not used for Armada 3700 Z1
SoC, and it is removed in Armada 3700 A0 SoC.
This patch removes the depleted command queue definition.

Change-Id: I5338d01322138e03131e4ed55ff5d7763d13aa30
Signed-off-by: Victor Gu <xigu@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/32442
Reviewed-by: Wilson Ding <dingwei@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/pci/host/pci-aardvark.c |   10 ----------
 1 file changed, 10 deletions(-)

diff --git a/drivers/pci/host/pci-aardvark.c b/drivers/pci/host/pci-aardvark.c
index 002ba24..2ed94d9 100644
--- a/drivers/pci/host/pci-aardvark.c
+++ b/drivers/pci/host/pci-aardvark.c
@@ -149,24 +149,14 @@
 #define CENTRAL_INT_BASE_ADDR			0x1b000
 #define HOST_CTRL_INT_STATUS_REG		(CENTRAL_INT_BASE_ADDR + 0x0)
 #define HOST_CTRL_INT_MASK_REG			(CENTRAL_INT_BASE_ADDR + 0x4)
-#define     PCIE_IRQ_CMDQ_INT			BIT(0)
 #define     PCIE_IRQ_MSI_STATUS_INT		BIT(1)
-#define     PCIE_IRQ_CMD_SENT_DONE		BIT(3)
 #define     PCIE_IRQ_DMA_INT			BIT(4)
-#define     PCIE_IRQ_IB_DXFERDONE		BIT(5)
 #define     PCIE_IRQ_OB_DXFERDONE		BIT(6)
 #define     PCIE_IRQ_OB_RXFERDONE		BIT(7)
-#define     PCIE_IRQ_COMPQ_INT			BIT(12)
-#define     PCIE_IRQ_DIR_RD_DDR_DET		BIT(13)
-#define     PCIE_IRQ_DIR_WR_DDR_DET		BIT(14)
 #define     PCIE_IRQ_CORE_INT			BIT(16)
 #define     PCIE_IRQ_CORE_INT_PIO		BIT(17)
 #define     PCIE_IRQ_DPMU_INT			BIT(18)
 #define     PCIE_IRQ_PCIE_MIS_INT		BIT(19)
-#define     PCIE_IRQ_MSI_INT1_DET		BIT(20)
-#define     PCIE_IRQ_MSI_INT2_DET		BIT(21)
-#define     PCIE_IRQ_RC_DBELL_DET		BIT(22)
-#define     PCIE_IRQ_EP_STATUS			BIT(23)
 #define     PCIE_IRQ_ALL_MASK			0xfff0fb
 #define     PCIE_IRQ_ENABLE_INTS_MASK		PCIE_IRQ_CORE_INT
 
-- 
1.7.9.5

